pytorch_lightning/utilities/distributed.py

```diff
@@ -360,15 +360,15 @@ def tpu_distributed() -> bool:
     return _TPU_AVAILABLE and xm.xrt_world_size() > 1
 
 
-def init_ddp_connection(
+def init_distributed_connection(
```

Comment:
1. this is a utility which could be used elsewhere. you'll need to provide backwards compatibility for `init_ddp_connection` - otherwise renaming this is a breaking change.
2. `init_dist_connection` ? slightly less verbose

Reply:
I didn't see valid use case of using or override init_ddp_connection() in internal fb. Is this BC mainly for who customize TTP?

Reply:
actually, this was added in https://github.com/PyTorchLightning/pytorch-lightning/pull/9044 which was targeted fro v1.5. meaning this code isn't available already, so there's no existing version to be backward compatible with